code/supervised/Logistic_Regression/logistic_ex1.py

```diff
@@ -1,17 +1,31 @@
-from sklearn.datasets import load_iris
-from sklearn.linear_model import LogisticRegression
-
-#Loads the sample dataset iris from sklearn's datasets
-X, y = load_iris(return_X_y=True)
-
-#Creates the Logistic Regression object
-clf = LogisticRegression(random_state=0, solver='lbfgs',multi_class='multinomial')
-#Fit the model to match the data
-clf = clf.fit(X, y)
-
-#Now, we show the predictions for each row of the dataset.
-print(clf.predict(X[:2, :]))
-print(clf.predict_proba(X[:2, :]))
-
-#Check the fit of the model.  Score returns the mean accuracy on the data.  
-print(clf.score(X, y))
+from sklearn.linear_model import LogisticRegression
+from sklearn.datasets import load_iris
+
+#For this example, we are using the iris dataset,
+#available with all sklearn distributions.
+#The parameter separates the return value, instead of returning an sklearn Bunch object
+X,y = load_iris(return_X_y = true)
+
+#Create the LR object for use with the dataset.
+#The fit function trains the model using the given data
+#the parameter random_state ensures that the results will
+#be the same every run.  Try different values if you want to verify
+#that there is nothing magic about this choice.
+model = LogisticRegression(random_state = 1).fit(X,y)
+
+#display the dataset being used
+print("----------------")
+print(X)
+print("----------------")
+print(y)
+print("----------------")
+#The predict function gives the class choice for each entry in the dataset
+print(model.predict(X[:2, :]))
```

Comment:
I'm not sure why you are predicting things you trained the model on. I would suggest creating some sample vectors [5.1 3.5 1.4 0.2]
 [4.9 3.  1.4 0.2]
 [4.7 3.2 1.3 0.2]
 [4.6 3.1 1.5 0.2]
 [5.  3.6 1.4 0.2]]
like this but change the values because that is part of the training set